pyproject.toml

```diff
@@ -1,21 +1,32 @@
-[tool.black]
+[tool.ruff]
 line-length = 120
-target-version = ["py37", "py38", "py39", "py310"]
-extend_exclude = '''
-/(
-    \.eggs
-  | \.git
-  | \.hg
-  | \.mypy_cache
-  | \.tox
-  | \.venv
-  | \.env
-  | _build
-  | buck-out
-  | build
-  | dist
-  | .vscode
-  | dev
-  | .coverage
-)/
-'''
\ No newline at end of file
+src = ["backend", "deploy"]
+
+
+[tool.ruff.lint]
+exclude = [ "backend/migrations","tests/**", "**/input_types.py", "**/mutations.py", "**/queries.py", "**/types.py", "**/api/__init__.py"]
```

Comment:
why exclude migrations?

Reply:
I enabled all formatting back. And added block-comment for current unused imports. In future, if something like this comes up again, we can decide on-the-go, if any other re-configuration is needed